waspc/data/Generator/templates/sdk/wasp/auth/forms/internal/common/LoginSignupForm.tsx

```diff
@@ -166,7 +166,12 @@ export const LoginSignupForm = ({
     onError: onErrorHandler,
     showEmailVerificationPending() {
       hookForm.reset()
-      setSuccessMessage(`You've signed up successfully! Check your email for the confirmation link.`)
+      setSuccessMessage(
+        `You've signed up successfully! Check your email for the confirmation link.`
+        {=# isEmailSenderProviderSetToDummy =}
+        + ` (NOTE(DEV): Your Wasp app is using Dummy email provider, which doesn't really send emails: instead, check server logs in the terminal for the email)`
```

Comment:
I like the idea, but I'm not a fan of this message.

I'm basing my feedback on the screenshot, so keep that in mind if something's changed since you created it.

Since this is something all users will see, I think we should make it as high-quality as reasonably possible.

Here's what I'd improve:
 - Make an entirely new message box (if that's easily done) that's in a different color and shows the dummy message.
 - If not, make the text less cluttered by removing the double parentheses and putting the dummy message in a separate new line (maybe even with a line in between)
 - Put the dummy email text in italics or something similar (slightly different color) to make it stand out as something a user wouldn't normally see (again, only if it's easily supported)
 - Improve the wording (by getting rid of filler words), fix grammatical errors (articles mostly), change "email" to "confirmation link" (possibly debatable). @vincanger can weigh in some more, he's a native speaker:

   ```suggestion
        + `NOTE(DEV): Your app is using the Dummy email provider, which doesn't really send emails. You can find the confirmation link in the server logs`
   ```

Reply:
> I think we should make it as high-quality as reasonably possible.

I agree with this point, I had the same reaction upon seeing the screenshot.

**This feels important to me:** You should be able to see what your users experience for real and you can write e2e tests that match the production UI. This extra piece of text feels off on those two points. 

Since this is all on the client, I'd rather go with a separate box or some sort of a design that indicates this is "dev mode" and then explain what that means. I don't have a complete look in my mind, but if you recall how Astro has it's dev toolbar - it's an _extra piece_ of UI on top of the _standard_ UI.

Reply:
>  I'd rather go with a separate box or some sort of a design that indicates this is "dev mode" and then explain what "dev mode" means. 

My first version of the comment suggested the same thing, but I didn't know how to precisely word it and feared it was too big to suggest :smile: 

But now that the cat's out of the bag, I can say I'm a big fan of this idea!